packages/client-api-schema/src/generated-schema.json

```diff
@@ -89,6 +89,9 @@
       "pattern": "^0x([a-fA-F0-9]{64})$",
       "type": "string"
     },
+    "ChallengeChannelError": {
```

Comment:
Is it safe to assume that generated-schema.json file does not need to be code reviewed?

Reply:
I suppose that depends on your level of paranoia 😄 I think it is OK to ignore in the code review, since this test with its improved coverage should pick up most cases of a mismatch between the types and the schema. We also now check that circle ci's git working tree is clean after running `yarn` -- which means we get an error if someone forgets to check in the new `generated-schema.json`.